infra/main.bicep

```diff
@@ -730,8 +685,8 @@ module privateEndpoints 'private-endpoints.bicep' = if (usePrivateEndpoint) {
     tags: tags
     resourceToken: resourceToken
     privateEndpointConnections: privateEndpointConnections
-    applicationInsightsId: monitoring.outputs.applicationInsightsId
-    logAnalyticsWorkspaceId: monitoring.outputs.logAnalyticsWorkspaceId
+    applicationInsightsId: useApplicationInsights ? monitoring.outputs.applicationInsightsId : ''
```

Comment:
This was missing a conditional since monitoring is an optional deployment gated on useApplicationInsights